service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseManagementOrderProgressionControllerTest.java

```diff
@@ -68,31 +96,88 @@ void aboutToSubmitShouldPopulateListServedCaseManagementOrdersWhenSendsToAllPart
             .build();
 
         CaseDetails caseDetails = CaseDetails.builder()
-            .data(caseDataMap(order, LocalDateTime.now().minusDays(1)))
+            .data(caseDataMap(order))
             .build();
 
-        AboutToStartOrSubmitCallbackResponse response = postAboutToSubmitEvent(buildCallbackRequest(caseDetails));
+        AboutToStartOrSubmitCallbackResponse response = postAboutToSubmitEvent(buildCallbackRequest(caseDetails,
+            ACTION_CASE_MANAGEMENT_ORDER));
 
         assertThat(response.getData())
             .containsOnlyKeys(SERVED_CASE_MANAGEMENT_ORDERS.getKey(), HEARING_DETAILS_KEY);
     }
 
-    private Map<String, Object> caseDataMap(CaseManagementOrder order, LocalDateTime localDateTime) {
+    @Test
+    void aboutToSubmitShouldBuildNotificationTemplateWhenSentToJudge() throws Exception {
+        CaseManagementOrder order = CaseManagementOrder.builder()
+            .status(SEND_TO_JUDGE)
+            .id(uuid)
+            .action(OrderAction.builder()
+                .type(SEND_TO_ALL_PARTIES)
+                .build())
+            .build();
+
+        CaseDetails caseDetails = CaseDetails.builder()
+            .data(ImmutableMap.<String, Object>builder()
+                .putAll(caseDataMap(order))
+                .putAll(getNotificationData())
+                .build())
+            .id(CASE_ID)
+            .build();
+
+        postAboutToSubmitEvent(buildCallbackRequest(caseDetails, DRAFT_CASE_MANAGEMENT_ORDER));
+
+        verify(notificationClient, times(1)).sendEmail(
+            eq(CMO_READY_FOR_JUDGE_REVIEW_NOTIFICATION_TEMPLATE), eq("admin@family-court.com"),
+            eq(expectedTemplateParameters()), eq(CASE_REFERENCE));
+    }
+
+    private Map<String, Object> caseDataMap(CaseManagementOrder order) {
         return ImmutableMap.of(
             CASE_MANAGEMENT_ORDER_JUDICIARY.getKey(), order,
             HEARING_DETAILS_KEY, List.of(Element.<HearingBooking>builder()
                 .id(uuid)
                 .value(HearingBooking.builder()
-                    .startDate(localDateTime)
+                    .startDate(YESTERDAY)
                     .build())
                 .build()));
     }
 
-    private CallbackRequest buildCallbackRequest(CaseDetails caseDetails) {
+    private Map<String, Object> getNotificationData() {
+        return ImmutableMap.of(
+            "cmoEventId", "draftCMO",
+            "caseLocalAuthority", LOCAL_AUTHORITY_CODE,
+            "familyManCaseNumber", FAMILY_MAN_CASE_NUMBER,
+            "respondents1", ImmutableList.of(
```

Comment:
why not `respondentLastName` going by the method name?

Reply:
`respondents1` is the property we store it on in CCD caseData